opentelemetry-semantic-conventions/scripts/generate-consts-from-spec.sh

```diff
@@ -35,7 +35,7 @@ docker run --rm \
 	-v "${CRATE_DIR}/scripts/templates:/templates" \
 	-v "${CRATE_DIR}/src:/output" \
 	otel/semconvgen:$SEMCOVGEN_VERSION \
-  --only span,event,scope \
```

Comment:
`scope` type is removed from latest build-tool, and is unused - https://github.com/open-telemetry/build-tools/pull/309